tests/integration/batch.js

```diff
@@ -361,41 +395,4 @@ describe('Batching', () => {
         });
     });
   });
-  describe('Slow requests', () => {
```

Comment:
Why remove these tests?

Reply:
They were to validate the `timeout` feature, which has been removed.